lib/poker_hands/CompareHands.rb

```diff
@@ -0,0 +1,5 @@
+class CompareHands
+  def call(hand1, hand2)
+    true
```

Comment:
Will this method always return a boolean? (Does it make sense for it to return a boolean?)

Reply:
In the case of the `determine_winner` method above, it returns the winner based on a true or false value, but to answer your question no, when you compare two hands you're not expecting a boolean, if it was `check_if_hand_wins(@hand1)` then you could expect a boolean.